openapi/v2/change/release_package.yml

```diff
@@ -0,0 +1,328 @@
+        required: false
+        schema:
+          type: boolean
+          format: boolean
+          default: false
+      responses:
+        '200':
+          description: Bill of Materials for a specified Item
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Assembly'
+        '404':
+          description: Change Order not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+components:
+  schemas:
+    ReleasePackage:
+      type: object
+      required:
+      - id
+      - name
+      properties:
+        Id:
+          type: integer
+          format: int64
+        Name:
+          type: string
+        affectedItems:
+          type: array
+          items:
+            $ref: '#/components/schemas/AffectedItem'
+        attachments:
+          type: array
+          items:
+            $ref: '#/components/schemas/Attachment'
+        next:
+          type: string
+          description: URL to retrieve the next page of affected items associated to this release package. Will only be provided if number of affected items exceeds provided limit or Propel limit of 1000
+    attributes:
+      type: object
+      required: 
+      - type
+      - url
+      properties:
+        type: 
+          type: string
+        url:
+          type: string
+    AffectedItem:
+      type: object
+      required:
+      - id
+      - name
+      properties:
+        Id:
+          type: string
+          description: Id of the associated Item Revision record
+        Name:
+          type: string
+          description: Name of the item record associated to the affected item
+        Custom_Field__c:
+          type: string
+          description: Placeholder for any custom fields an org may have on the Change__c object.
+        bom:
+          type: array
+          items:
+            $ref: '#/components/schemas/Assembly'
+        aml:
+          type: array
+          items:
+            $ref: '#/components/schemas/Manufacturer'
+        attachments:
+          type: array
+          items:
+            $ref: '#/components/schemas/Document'
+    Assembly:
+      type: object
+      description: Assembly row represented in a BOM. 
+      required:
+      - id
+      - name
+      - level
+      properties:
+        Id:
+          type: string
+          description: Id of the child Item record
+        PDLM__Item_Number__c:
+          type: string
+          description: Name of the child Item record
+        PDLM__Level__c:
+          type: string
+          description: 'BOM level of assembly row. In the format of 1, 1.1, 2, 2.1, 2.2, 2.2.1'
+        Custom_Field__c:
+          type: string
+          description: Placeholder for any custom fields an org may have on the Assembly__c object.
+    Attachment:
```

Comment:
is this just ContentDocument?

Reply:
Its a content version record. figured the best way to get the content version Id, plus the contentDocumentId.